dv/riscv_compliance/ibex_riscv_compliance.core

```diff
@@ -34,6 +34,11 @@ parameters:
     paramtype: vlogparam
     default: false
     description: Enable the E ISA extension (reduced register set)
+  RV32B:
+    datatype: bool
+    paramtype: vlogparam
+    default: false
+    description: Enable the B ISA extension (bit manipulation EXPERIMENTAL) [0/1]
```

Comment:
The 0/1 is wrong/confusing, please remove it.

I know this goes slightly beyond this PR, but can you align the descriptions of these parameters between the files? `ibex_simple_system.core` has the `[0/1]` e.g. for the RV32E parameter, which should be removed as well.

Reply:
Ah whoops, copy/paste issue.

For this PR I was just aiming to match the existing style across .core files. We can clean up as part of https://github.com/lowRISC/ibex/issues/753